### PR TITLE
Update conf.py template

### PR DESCRIPTION
**Pull request by pdavide**

### CODE REVIEW

File: readthedocs/templates/doc_builder/conf.py.tmpl [hunk 1]
```diff
@@ -38,13 +38,23 @@ if os.path.exists('_static'):
...
-# Overriding the theme with the docs italia one
+# Setting the theme to docs-italia-theme
 theme = __import__('docs-italia-theme')
 html_theme = 'docs-italia-theme'
 html_style = None
 html_theme_options = {}
 html_theme_path = [theme.get_html_theme_path()]
 
+# docs-italia-theme default options
+html_title = u'{{ project.name }}
```
Comment (xrmx) [inline readthedocs/templates/doc_builder/conf.py.tmpl]: missing '

File: readthedocs/templates/doc_builder/conf.py.tmpl [hunk 2]
```diff
@@ -38,13 +38,23 @@ if os.path.exists('_static'):
...
+html_use_index = False
+numfig = True
+latex_documents = [
+  ('index', {{ project.slug }} + '.tex', u'{{ project.name }}, u'{{ project.publisherproject.first.publisher.name }}, 'manual'),
```
Comment (xrmx) [inline readthedocs/templates/doc_builder/conf.py.tmpl]: strings must be enclosed with '

File: readthedocs/templates/doc_builder/conf.py.tmpl [hunk 3]
```diff
@@ -38,13 +38,23 @@ if os.path.exists('_static'):
...
+]
+latex_show_urls = 'footnote
```
Comment (xrmx) [inline readthedocs/templates/doc_builder/conf.py.tmpl]: missing '

**Review by xrmx — CHANGES_REQUESTED** [review]

**Review by xrmx — COMMENTED** [review]
If you can please squash the two commit it's fine for me

**Review by yakky — APPROVED** [review]

**Comment by yakky** [comment]
@xrmx is this ok for you now?

**Review by xrmx — APPROVED** [review]

**Pull request merged**